src/main/java/com/edmunds/tools/databricks/maven/BaseDatabricksMojo.java

```diff
@@ -182,6 +192,13 @@ void setProject(MavenProject project) {
         this.project = project;
     }
 
+    /**
+     * NOTE - only for unit testing!
+     */
+    void setPrefixToStrip(String prefixToStrip) {
```

Comment:
ok so with test harness, this is no longer necessary. Personally, I find using test-harness to set these properties better as it is a more complete test. The negatives is that it is more verbose in other ways and without standards, it could be more confusing.

Reply:
ok, thanks. I'll try it.